### DISCUSSION TITLE
Dynamic rows and full height

### DISCUSSION THREAD

**Discussion by TmLev** [TanStack/virtual#338] [Q&A] [answered]
How can one change [example with dynamic size rows](https://tanstack.com/virtual/v3/docs/examples/react/dynamic) so that parent container expands in a way that makes whole page scrollable? It's similar to rendering all rows, but keeping rows offscreen virtualised at the same time.

**Comment by TmLev** [comment, marked as answer]
I've got it working with `useWindowVirtualizer`.

**Reply by jkinley** [reply]
@TmLev Any chance you could post an example?

**Reply by jkinley** [reply]
I still have not been able to get this working as expected. Has anyone else figured out how to do this?

**Reply by TmLev** [reply]
@jkinley, sorry, had a lot of work this week. 

Does this help? https://codesandbox.io/s/priceless-browser-w4u6j3?file=/src/App.tsx
